draft-ietf-quic-recovery.md

```diff
@@ -769,7 +769,10 @@ Implementations MUST either use pacing or limit such bursts to the initial
 congestion window, which is recommended to be the minimum of
 10 * max_datagram_size and max(2* max_datagram_size, 14720)), where
 max_datagram_size is the current maximum size of a datagram for the connection,
-not including UDP or IP overhead.
+not including UDP or IP overhead. An implementation that does not use pacing
+SHOULD limit the congestion window increase during slow start to
```

Comment:
In order to meet the previous criteria, how about "An implementation that doest not use pacing MAY limit the congestion window increase during slow start to 2 * max_datagram_size_ per ACK frame received {{?RFC3465}} in order to help limit bursts to the  initial congestion window."

Reply:
I think we can say either "MAY" or "SHOULD" instead of MAY/SHOULD. I am ok with changing it to "MAY" as there is already a requirement for limiting it to 10 datagrams in the previous line.

Reply:
We've already said that a sender MUST limit bursts to IW, so there's no reason to repeat that. I would argue this is a SHOULD-level requirement; MAY is strange, since that's really a permission and that's not what we're going after here. I'd suggest dropping the phrase "in order to limit bursts" --- "An implementation that does not use pacing SHOULD limit the congestion window increase during slow start to 2 * max_datagram_size per ACK frame received {{?RFC3465}}. "

Reply:
sounds good to me. Updated it